test/joiner_join_project_data_test.rb

```diff
@@ -25,7 +25,8 @@ def test_join_project
         { 'msb-usa' =>
           {
             'name' => 'MSB-USA', 'status' => 'Hold',
-            'self' => project_self_link('msb-usa')
+            'self' => project_self_link('msb-usa'),
+            'categories' => []
```

Comment:
Ugh. Jekyll 3.0.0 `Document.new` adds that. 3.0.0pre-beta2 does not. Should we update the test to account for `categories` or lock down the jekyll version in the Gemfile?